packages/sdk/src/sdk/api/tracks/TracksApi.ts

```diff
@@ -661,4 +705,13 @@ export class TracksApi extends GeneratedTracksApi {
     }
     return this.solanaClient.sendTransaction(transaction)
   }
+
+  /**
+   * Generates a new track ID
+   *
+   * @hidden
+   */
+  async generateTrackId() {
```

Comment:
public? I have diverged by making the user one private :-D

Reply:
Yup! We use it in the upload sagas